src/components/Content/Counter/Ingredient/Ingredient.js

```diff
@@ -0,0 +1,74 @@
+import './Ingredient.css';
+import React from "react";
+
+class Ingredient extends React.Component {
+
+    constructor (props) {
+        super(props);
+
+        this.state = {
+            ingredient: 0,
+            disabledMinus: true,
+            disabledPlus: false
+        }
+    }
+
+    addOne () {
+        this.setState((oldState) => {
+            if (this.state.ingredient === 4) {
+                return {
+                    ...oldState,
+                    ingredient: oldState.ingredient + 1,
+                    disabledPlus: !oldState.disabledPlus
+                }
+            } else {
+                return {
+                    ...oldState,
+                    ingredient: oldState.ingredient + 1,
+                    disabledMinus: false
+                }
+            }            
+        }) 
+    }
+
+    removeOne () {
+        this.setState((oldState) => {
+            if (this.state.ingredient === 5) {
+                return {
+                    ...oldState,
+                    ingredient: oldState.ingredient - 1,
+                    disabledPlus: !oldState.disabledPlus
+                }
+            } else if (this.state.ingredient === 1) {
+                return {
+                    ...oldState,
+                    ingredient: oldState.ingredient - 1,
+                    disabledMinus: !oldState.disabledMinus 
+                }
+            } else {
+                return {
+                    ...oldState,
+                    ingredient: oldState.ingredient - 1 
+                }
+            }
+         }) 
+    }
+
+    render() {
+        return (
+            <div className='counter__ingredient'>
+                <button onClick={(sum) => {
+                        this.addOne();
+                        this.props.func(this.props.priceOfIngredient);
+                    }} disabled={this.state.disabledPlus}>&#43;</button>
+                <button onClick={(sum) => {
+                        this.removeOne();
```

Comment:
This is not good-looking code; `onClick` needs to call a single function that can be understandable. But here, instead, you are adding functionality to the `onClick` callback. Keep JSX and functions separate. Just group function calls outside `onClick` etc.